code/modules/shuttle/marine_dropship.dm

```diff
@@ -661,6 +676,7 @@
 	to_chat(user, span_danger("A loud alarm erupts from [src]! The fleshy hosts must know that you can access it!"))
 	user.hive.on_shuttle_hijack(crashing_dropship)
 	playsound(src, 'sound/misc/queen_alarm.ogg')
+	crashing_dropship.prevent_silicon_intervention()
```

Comment:
yes, but you should do when the xeno touch the console or unlock all doors as well.